tests/account_keytype_test.go

```diff
@@ -324,6 +324,15 @@ func generateDefaultTx(sender *TestAccountType, recipient *TestAccountType, txTy
 		values[types.TxValueKeyAnchoredData] = dataAnchor
 		values[types.TxValueKeyFeePayer] = recipient.Addr
 		values[types.TxValueKeyFeeRatioOfFeePayer] = ratio
+	case types.TxTypeAccessList:
+		values[types.TxValueKeyNonce] = sender.Nonce
+		values[types.TxValueKeyTo] = recipient.Addr
+		values[types.TxValueKeyAmount] = amount
+		values[types.TxValueKeyGasLimit] = gasLimit
+		values[types.TxValueKeyGasPrice] = gasPrice
+		values[types.TxValueChainID] = big.NewInt(1)
```

Comment:
```suggestion
		values[types.TxValueChainID] = big.NewInt(8217)
```

Reply:
The blockchain instance generated by NewBCData() which used chain id of signer has a chain ID of 1. The chain id of signer and AccessList Tx must have the same value. 
Do we need to change this value?